src/hooks/UseNewID.js

```diff
@@ -1,10 +1,9 @@
-import { useMemo } from 'react';
-
-let lastID = 1000;
+import { useMemo, useState } from 'react';
 
 const useNewID = () => {
+  const [lastID, setLastID] = useState(1000);
```

Comment:
My change here is wrong (as pointed out by the linter). 
The problem why I changed it was, that on the server side render and the client hydration differnt IDs where generated I think.